### ISSUE TITLE
Invalid signature. Expected basestring is GET&http%3A%2F%2Fapi.fanfou.com

### ISSUE THREAD

**Issue by browncrane** [LitoMore/nofan#58]
After config consumer key and login success.  
shows `Invalid signature. Expected basestring is GET&http%3A%2F%2Fapi.fanfou.com`
When running `nofan`
On Ubuntu 18.04 Desktop 64bit, using npm 3.5.2.

**Comment by LitoMore** [comment]
Hi @browncrane 

Could you provide all the output of the `nofan config -a`? Your consumer key and consumer secret are not required.

**Comment by browncrane** [comment]
Hi @LitoMore. Thanks for the quick response.
After run `nofan config -a`, find there are more staus to be config. It works after  I config them all.
~If I remember correctly,~  it exits after I chose the `Global Settings` made some selection and press `Enter`.
Double checked this, with reinstall it.
Terminal looks like this.
```
~$ nofan config
? Enter your consumer key 2d9*****70baea44
? Enter your consumer secret 51****3f
? How many statuses would you like to display (1 - 60) 10
? Global Settings photo_tag, use_https
me@mymachine:~$
```

**Comment by LitoMore** [comment]
OK.

There is a bug in `Nofan._handleError()` function, it should have a config tip when the base string is not the expected one.

Fixed on e1970555c0368c5687581cc8425a62cc58068064. Also, a new version released.

Closing.
